include/JSystem/JUT/JUTProcBar.h

```diff
@@ -7,13 +7,61 @@
 class JUTProcBar {
 public:
 	struct CTime {
-		CTime();
+		u32 startTick;
```

Comment:
Use proper naming schemes please. e.g. like 'm_example' or 'mExample'

Reply:
going forward, we'll be using mExample consistently